cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/resources/test-authz-provider.ini

```diff
@@ -84,3 +84,6 @@ readers_all_ns3_ds_type = instance=cdap->namespace=ns3->dataset_type=table??->ac
 all_ns3_ds_mod = instance=cdap->namespace=ns3->dataset_module=tab_*_mod->action=*
 ; secure key
 admin_ns3_key = instance=cdap->namespace=ns3->securekey=*_key-??->action=admin
+; principal
+admin_alice = instance=cdap->principal=alice->action=admin
```

Comment:
lets put kerberos principal sample like 
alice/host@host.net
etc to make sure privilege which are stored as text don't get messed up